runtimes/common/src/migrations.rs

```diff
@@ -0,0 +1,77 @@
+// KILT Blockchain – https://botlabs.org
+// Copyright (C) 2019-2022 BOTLabs GmbH
+
+// The KILT Blockchain is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+
+// The KILT Blockchain is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+
+// You should have received a copy of the GNU General Public License
+// along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+// If you feel like getting in touch with us, you can do so at info@botlabs.org
+
+use core::marker::PhantomData;
+use frame_support::traits::Get;
+use hex_literal::hex;
+
+pub struct RemoveKiltLaunch<R>(PhantomData<R>);
+impl<R: frame_system::Config> frame_support::traits::OnRuntimeUpgrade for RemoveKiltLaunch<R> {
+	fn on_runtime_upgrade() -> frame_support::weights::Weight {
+		let items = match frame_support::storage::unhashed::kill_prefix(&hex!("37be294ab4b5aa76f1df3f80e7c180ef"), None)
```

Comment:
Can this be generated from the pallet name string?

Reply:
I wanted to do it this way but since we are removing the pallet from the runtime, we can't derive it's pallet name from the storage. You can verify the hex yourself via the Polkadot Apps
<img width="1131" alt="Screenshot 2022-04-28 at 10 22 03" src="https://user-images.githubusercontent.com/26296386/165709976-66ce9bd6-49e9-4c4a-8acc-7a46b7612815.png">

Reply:
Was just playing around, and this is what apparently works: `let hash = sp_io::hashing::twox_128(b"KiltLaunch");`.

No need to change it, but I think it's better for readability and for potentially fix any issues we identify in the future.